src/run/headers.ts

```diff
@@ -54,7 +54,10 @@ const generateNetlifyVaryValues = ({
 }
 
 const getHeaderValueArray = (header: string): string[] => {
-  return header.split(',').map((value) => value.trim())
+  return header
+    .split(',')
+    .map((value) => value.trim())
+    .filter(Boolean)
```

Comment:
https://github.com/vercel/next.js/pull/71159/files#diff-ae41ee59d88093c9c0551e12f654b1b99e4211b1c9fa6dc1471febb0361004e7R21 this can now produce invalid value like `s-maxage=31536000, ` so we remove ~falsy/empty directives, so at least on our end we produce valid value.

Hopefully they fix their stuff in which this added `.filter` would do nothing (so it also won't hurt to have it always)